### ISSUE TITLE
Automatic upload of the final result to an FTP server.

### ISSUE THREAD

**Issue by Thiagobhmg** [MattMcManis/Axiom#53]
Add automatic upload functionality of the final result to an FTP server.

Since all execution is via shell, you could add sending the final result via FTP, this would be very useful.

Even file upload functionality for Amazon S3 servers and other services.

@ncftp

The commands are very similar to Windows ftp, to connect use the following command:

`ncftp -u user -p password ftp.site.com`

Directory with files or final converted file

`mput -r directory-with-files`

**Comment by MattMcManis** [comment]
Yes, it is possible I could add FTP type, username, and password fields in the Settings Tab.

I will let you know.
